pkg/knuu/instance_helper.go

```diff
@@ -166,7 +166,8 @@ func (i *Instance) deployPod() error {
 // destroyPod destroys the pod for the instance
 // Skips if the pod is already destroyed
 func (i *Instance) destroyPod() error {
-	err := k8s.DeleteStatefulSet(k8s.Namespace(), i.k8sName)
+	grace := int64(0)
```

Comment:
we need a comment for ourselves in the func that now it's a force delete with no grace period